tgui/packages/tgui/interfaces/MarineCasship.tsx

```diff
@@ -60,7 +61,7 @@ export const MarineCasship = (props, context) => {
           if (keyCode === KEY_6) {
             act('deselect');
           }
-          if (data.plane_state !== 0) {
+          if ((data.plane_state !== 0) && (limiter === false)) {
```

Comment:
Wouldnt just checking if the directions are the same be better? 
You wouldnt miss rotations randomly and it wouldnt topic spam

Reply:
Yeah I'm dumb, will change it tomorrow